valohai/internals/api_calls.py

```diff
@@ -0,0 +1,47 @@
+import json
+from typing import TYPE_CHECKING, Any, Dict
+
+from valohai import paths
+
+if TYPE_CHECKING:
+    import requests
+
+
+def get_api_requests_kwargs(endpoint: str) -> Dict[str, Any]:
+    """
+    Get the "presigned call" dict for a given endpoint from the
+    API JSON configuration file.  Will happily throw all sorts of
+    exceptions e.g. if the API JSON file is missing or malformed.
+    """
+    with open(paths.get_api_config_path()) as json_file:
+        api_config = json.load(json_file)
+    value = api_config.get(endpoint)
+    if not (isinstance(value, dict) and value.get("url")):
+        raise ValueError(f"Invalid API config for {endpoint}")
```

Comment:
some sanity checking

not apparent that the dict will also contain other important stuff but should be fine